sample/development/components/Screens/NewScreen.xml

```diff
@@ -68,7 +68,6 @@ governing permissions and limitations under the License.-->
   <script type="text/brightscript" uri="pkg:/source/adobe/core/identityState.brs"/>
   <script type="text/brightscript" uri="pkg:/source/adobe/edge/edgeModule.brs"/>
   <script type="text/brightscript" uri="pkg:/source/adobe/edge/edgeRequest.brs"/>
```

Comment:
I notice adding or removing a file requires changes to lot of other files. Can you add a task to automate this?

Reply:
Sure